python/tests/test_dwelling.py

```diff
@@ -90,6 +112,25 @@ def test_from_code(self):
     def test_from_unknown_code(self):
         assert dwelling.Region.from_data('CA') == dwelling.Region.UNKNOWN
 
+class TestCeiling:
+
+    @pytest.fixture
+    def sample(self):
```

Comment:
This same example is used in `sample_input_d` as well. Define in one place and use in both?